lib/widgets/favourite_tile.dart

```diff
@@ -0,0 +1,99 @@
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:we_find/model/model_wrapped.dart';
+import 'package:we_find/providers/fav_course_provider.dart';
+import 'package:we_find/screens/course_detail_screen.dart';
+
+import '../model/model.dart';
```

Comment:
nitpick but `import 'package:we_find/model/model.dart' ` is more consistent with the rest of the imports